ingenialink/ethercat/servo.py

```diff
@@ -50,8 +50,9 @@ class EthercatServo(PDOServo):
     DICTIONARY_CLASS = EthercatDictionary
     MAX_WRITE_SIZE = CAN_MAX_WRITE_SIZE
 
+    DEFAULT_WORKING_COUNTER = 0
```

Comment:
I think default is not a good name if it means that something fail

Reply:
When the master sends an EtherCAT frame it sets the working counter to zero, If this value is unchanged in the reply then it means that something went wrong. I think that the name is correct. The error message is also clear.

Reply:
If someone doesn't know what working counter 0 means and reads this name will think that 0 is good. We must write explicitly that 0 means error or fail in this case.